.husky/pre-push

```diff
@@ -0,0 +1 @@
+npm test -- --run
```

Comment:
Seems to me that this should happen before you commit something that may be breaking the test